migrations/versions/96a8d065e5ea_.py

```diff
@@ -18,6 +18,38 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('campaigns',
```

Comment:
the master migration should create all tables `campaigns, requests, contributors, and campaign_contributors`